packages/react/src/themes/teams/components/Dropdown/dropdownItemStyles.ts

```diff
@@ -17,13 +20,51 @@ const dropdownItemStyles: ComponentSlotStylesInput<DropdownItemProps, DropdownVa
       ...getBorderFocusStyles({
         siteVariables,
         isFromKeyboard: p.isFromKeyboard,
+        borderRadius: 0,
       })[':focus'],
       ...(!p.isFromKeyboard && {
         color: v.listItemColorHover,
         backgroundColor: v.listItemBackgroundColorHover,
+        ...(p.header && {
+          [`& .${DropdownItem.slotClassNames.header}`]: {
+            color: v.listItemColorHover,
+          },
+        }),
+        ...(p.content && {
+          [`& .${DropdownItem.slotClassNames.content}`]: {
+            color: v.listItemColorHover,
+          },
+        }),
       }),
     }),
   }),
+  image: ({ props: p }): ICSSInJSStyle => ({
+    margin: `${pxToRem(3)} ${pxToRem(12)} ${pxToRem(3)} ${pxToRem(4)}`,
+    // using !important because:
+    // a ' > :not(:last-child)' selector affecting the margin-right style
+    marginRight: `${pxToRem(12)} !important`,
```

Comment:
The image is being affected by a selector
`'> *:not(:last-child)'`
Which is applying an incorrect margin-right value. It is taking precedence over my style. It may be coming from flexStyles?